.github/workflows/close-stale-issues.yml

```diff
@@ -0,0 +1,30 @@
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 14
```

Comment:
14 days look quite short and would probably comment on 300+ issues. Can we first extend to smth like 360 and decrease in follow-up?